distribution/docs/src/main/resources/content/_running/available-console-commands.adoc

```diff
@@ -0,0 +1,50 @@
+:title: Available Console Commands
+:type: maintaining
+:status: published
+:parent: Console Commands
+:summary: Types of console commands available.
+:order: 02
+
+== {title}
+
+${branding} provides several types of console commands for administrators to use to perform tasks.
+Additionally, many default Karaf commands are also enabled.
+For more information about Karaf commands, see the http://karaf.apache.org/documentation.html[Karaf documentation].
+
+For a complete list of all available commands, from the ${command-console}, press *TAB* and confirm when prompted.
+
+Console commands follow a format of `namespace:command`.
+
+To get a list of commands, type in the namespace of the desired extension then press *TAB*.
+
+For example, type `catalog`, then press *Tab*.
+
+.[[available_console_commands]]${branding} Console Command Namespaces
+[cols="3,1,5" options="header"]
+|===
+|Title
```

Comment:
I don't think this column is necessary. It doesn't provide any more value than the Namespace column.

Reply:
fixed